frontend/src/components/Header.tsx

```diff
@@ -146,6 +150,28 @@ const Header = ({ logoPath, locale }: Props) => {
   const title =
     usePathname() === "/" ? t("title") : <Link href="/">{t("title")}</Link>;
 
+  const LoginLink = () => {
+    return (
+      <Link
+        href={process.env.auth_login_url as string}
+        key={process.env.auth_login_url as string}
+        className={clsx({
```

Comment:
if all of these are always true we probably don't need to use `clsx` here

Reply:
Updated, thanks.